app/models/manageiq/providers/openshift/container_manager/refresh_parser.rb

```diff
@@ -165,10 +167,11 @@ def parse_env_variables(env_variables)
         end
       end
 
-      def parse_openshift_image(openshift_image)
+      def parse_openshift_image(openshift_image, options)
         id = openshift_image[:dockerImageReference] || openshift_image[:metadata][:name]
         ref = "#{ContainerImage::DOCKER_PULLABLE_PREFIX}#{id}"
-        new_result = parse_container_image(id, ref)
+        new_result = parse_container_image(id, ref, :store_new_images => options.store_unused_images)
```

Comment:
My thinking was that this function is called from two places (pods, images), with potentially different value.  it doesn't have the context to understand which images are unused, only whether it already has it in data_index.
Even when setting is false, while parsing pods we want to add "new" images. IMO it'd be confusing to pass `store_unused_images: true` when the eponymous setting is false.